repos/system_upgrade/common/actors/commonleappdracutmodules/files/dracut/85sys-upgrade-redhat/do-upgrade.sh

```diff
@@ -196,8 +196,8 @@ ibdmp() {
 
 bring_up_network() {
     if [ -f /etc/leapp-initram-network-manager ]; then
-	# NOTE(ivasilev) Reverting the change to see if it caused the crash
-	. /lib/dracut/hooks/cmdline/99-nm-config.sh
+    # NOTE(ivasilev) Reverting the change to see if it caused the crash
```

Comment:
```suggestion
        # NOTE(ivasilev) Reverting the change to see if it caused the crash
```

Reply:
Actually, I believe the comment can be removed as well - iirc it's a left-over of trying to apply pvalena's suggestions in initram network pr https://github.com/oamg/leapp-repository/pull/960/files#diff-bcf4627581aafbb8e8864d36b3e5c6a58399c5814ff0e7ff26589e94f5a998caR197

Reply:
oki. dropping